ClubKey/ClubKey.Domain/Repositories/Implementations/ClubRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using ClubKey.Data.Entities;
+using ClubKey.Data.Entities.Models;
+using ClubKey.Domain.Repositories.Interfaces;
+
+namespace ClubKey.Domain.Repositories.Implementations
+{
+    public class ClubRepository : IClubRepository
+    {
+        public ClubRepository(ClubKeyContext context)
+        {
+            _context = context;
+        }
+        private readonly ClubKeyContext _context;
+        public List<Club> GetClubsByCityId(int cityId)
```

Comment:
I think we'll be getting cities by "location" and not by cityId but can't say for sure